inc/update.class.php

```diff
@@ -496,6 +496,11 @@ public function doUpdates($current_version = null) {
             update955to956();
             break;
```

Comment:
Next upgrade will not be done if made from a version prior to 9.5.6.
```suggestion
```

Reply:
Hi.
Thanks for the tip, the tests are now working.